### PR TITLE
Arxiv malformed

### PR DESCRIPTION
**Pull request by brucemiller**
This PR fixes a variety of problems revealed as Error/Malformed in the recent arXiv run; typically things like `ltx:section` appearing out of place due to mismanagement of lists, itemizations, bibliographies, tables and such.

Still some tempting candidates to look at ...

### CODE REVIEW

File: lib/LaTeXML/Package/LaTeX.pool.ltxml [hunk 1]
```diff
@@ -4029,7 +4033,8 @@ DefConstructor('\lx@mung@bibliography@pre', sub {
     if ($tag =~ /^ltx:(?:itemize|enumerate|description)$/) {
       $document->maybeCloseElement($tag); }    # Or even remove (if empty)?
     return; });
-DefConstructorI('\lx@bibnewblock', undef, "<ltx:bibblock>");
+DefConstructorI('\lx@bibnewblock', undef, sub {
+    $_[0]->openElement('ltx:bibblock') if $_[0]->isOpenable('ltx:bibblock'); });
```
Comment (dginev) [inline lib/LaTeXML/Package/LaTeX.pool.ltxml:4037]: shouldn't we open _something_ when a block is not openable? Just to keep the contents grouped. Maybe a generic block or inline-block ?

**Comment by brucemiller** [comment]
Might as well call this done, unless I find something new before you want it merged.

File: lib/LaTeXML/Package/LaTeX.pool.ltxml (hunk 1 above)
Comment (brucemiller) [reply lib/LaTeXML/Package/LaTeX.pool.ltxml:4037]: The scenario seems to be that we've somehow already gotten into a bibitem and block when this is needed.

Comment (dginev) [reply lib/LaTeXML/Package/LaTeX.pool.ltxml:4037]: Oh, great then, nevermind 👍🏻

File: t/babel/numprints.xml [hunk 2]
```diff
@@ -1352,19 +1352,14 @@ vs.
                 </XMath>
               </Math></td>
           </tr>
-          <tr>
-            <td align="char:." thead="row">before  after</td>
```
Comment (dginev) [inline t/babel/numprints.xml:1356]: ouch, the test was wrong here? Interesting find.

**Review by dginev — APPROVED** [review]
Nice, everything looks worth having! 

The `\lx@NAT@parselabel` looks a bit scary, but we can refine it after we hit arXiv with it and see what happens.

**Comment by brucemiller** [comment]
> The ***@***.***@parselabel| looks a bit scary, 

Indeed; the biggest delay was convincing myself to code it,
rather than to figure out what the TeX version was even *trying* to do!

**Pull request merged**